fields/types/localfiles/LocalFilesField.js

```diff
@@ -37,9 +37,8 @@ var LocalFilesFieldItem = React.createClass({
 	},
 
 	render () {
-		let { filename } = this.props;
-		let ext = filename.split('.').pop();
-
+		const { filename, originalname, itemId, listKey } = this.props;
+		const isPicture = originalname && originalname.match(/\.(jpeg|jpg|gif|png)$/) != null;
```

Comment:
This should probably based on filetype instead of originalname. I'd test filetype first, and if it's `application/octet-stream`, test originalname.